.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -61,6 +71,7 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       branch: ${{ steps.extract-inputs.outputs.branch }}
+      fully-specified-branch: refs/heads/${{ steps.extract-inputs.outputs.branch }}
```

Comment:
There is one place that wants `branch` without `ref/heads`, so I added another variable to make conditional statements easier.